docs/source/AdministratorGuide/Systems/DataManagement/index.rst

```diff
@@ -1,351 +1,19 @@
-
-- Now you need to make sure that the DIRAC components that take care of FTS transfers are in place. You need to configure and startup a number of components. This can be done with the "dirac-setup-server" command and a the following FTS.cfg describing what you need:
-
-::
-
-      LocalInstallation
-      {
-        Systems = DataManagement, RequestManagement
-        DataBases = RequestDB
-        Services = DataManagement/TransferDBMonitoring
-        Agents = DataManagement/FTSSubmitAgent, DataManagement/FTSMonitorAgent
-      }
-
-- Then one needs to configure the DIRAC Channels that will be handled by the FTS Agents. The methods to create and manipulate the DIRAC Channels for FTS are not exposed on a Service interface. This has to be done with a simple python script from the server:
-
-::
-
-   from DIRAC.Core.Base import Script
-   Script.parseCommandLine()
-   from DIRAC.DataManagementSystem.DB.TransferDB import TransferDB
-
-   sourceSite = 'ShortSite-Name1'         # LCG.CERN.ch -> CERN
-   destinationSite = 'ShortSite-Name2'
-
-   transferDB = TransferDB()
-
-   res = transferDB.createChannel( sourceSite, destinationSite )
-   if not res['OK']:
-     print res['Message']
-     exit(-1)
-
-   channelID = res['Value']
-   print 'Created FTS Channel %s' % channelID
-
-- At this point some transfer can be attempted between the configured SEs. For that purpose you can use the command line script:
-
-::
-
- $ dirac-dms-fts-submit -h
-   Submit an FTS request, monitor the execution until it completes
- Usage:
-   dirac-dms-fts-submit [option|cfgfile] ... LFN sourceSE targetSE
- Arguments:
-   LFN:      Logical File Name or file containing LFNs
-   sourceSE: Valid DIRAC SE
-   targetSE: Valid DIRAC SE
- General options:
-   -o:  --option=         : Option=value to add
-   -s:  --section=        : Set base section for relative parsed options
-   -c:  --cert=           : Use server certificate to connect to Core Services
-   -d   --debug           : Set debug mode (-dd is extra debug)
-   -h   --help            : Shows this help
-
-::
-
-  $ dirac-dms-fts-submit /lhcb/user/r/rgracian/fts_test CNAF-USER PIC-USER
-  Submitted b3c7c25a-1d14-11e1-abe9-dc229ac9908c @ https://fts.pic.es:8443/glite-data-transfer-fts/services/FileTransfer
-  |====================================================================================================>| 100.0% Finished
-
-
-Using this script, the request to the FTS server will be formulated following the information configured in DIRAC, and will be submitted form your client to the selected FTS server with your local credential. Make sure you are using a proxy that is authorized at your FTS server (usually only some specific users in the VO are allowed, contact the administrators of the site offering you this server in case of doubts).
+.. toctree::
+   :maxdepth: 1
 
-.. include:: agents.rst
-.. include:: data_logging_system.rst
+   concepts
+   dfc
+   fts
+   agents
```

Comment:
This last one looks... dubious to me. Is it this? https://github.com/DIRACGrid/DIRAC/blob/master/docs/source/AdministratorGuide/Systems/DataManagement/agents.rst

the above has been removed from the integration branch.

Reply:
Sh*** I did not remember I changed the doc already in integration. Maybe I should rebase on integration all these changes ?